src/main/java/frc5124/robot2020/RobotContainer.java

```diff
@@ -108,12 +127,15 @@ private void configureButtonBindings(){
     operatorDown.whileHeld(new LiftDown(hanger));   
     operatorRB.whileHeld(new RotateTurret(turret, RobotMap.TurretMap.turretSpeed));
     operatorLB.whileHeld(new RotateTurret(turret, -RobotMap.TurretMap.turretSpeed));
-    //operatorUp.whenPressed(new SetShootVelocity(shooter, RobotMap.ShooterMap.shootVelocity));
-
+    operatorRight.whenPressed(new SetShootRPM(shooter, RobotMap.ShooterMap.lineRefRPM));
```

Comment:
don't pass a constant from the button binding to the command. the subsystem should know which constant to use